module/commands/regolamento_didattico.py

```diff
@@ -125,8 +129,10 @@ def get_reg_keyboard(reg_doc: dict = None) -> InlineKeyboardMarkup:
             InlineKeyboardButton('L-31', callback_data='reg_button_triennale_L31'),
             InlineKeyboardButton('LM-18', callback_data='reg_button_magistrale_LM18'),
             InlineKeyboardButton('L-35', callback_data='reg_button_triennale_L35'),
-            InlineKeyboardButton('LM-40', callback_data='reg_button_magistrale_LM40'),
-        ]])
+            InlineKeyboardButton('LM-40', callback_data='reg_button_magistrale_LM40')],
+           [InlineKeyboardButton('Menu principale  ⏪', callback_data='reg_button_help')]
+        ])
     keyboard = [[InlineKeyboardButton(r.split('_')[0], callback_data=r)] for r in reg_doc]
-    keyboard.append([InlineKeyboardButton('Indietro', callback_data='reg_button_home')])  # back button
+    keyboard.append([InlineKeyboardButton('Indietro  ◀️', callback_data='reg_button_home')])  # back button
+    keyboard.append([InlineKeyboardButton('Menu principale  ⏪', callback_data='reg_button_help')]) # menu button
```

Comment:
to avoid redundant code put `[InlineKeyboardButton('Menu principale  ⏪', callback_data='reg_button_help')` in a separate variable like `BACK_TO_MENU`